src/inline.tsx

```diff
@@ -43,6 +43,7 @@ type Props = {
     children: React.ReactNode;
     className?: string;
     role?: string;
+    'aria-label'?: string;
```

Comment:
I'm not sure why we weren't allowing aria-label for `Inline` and `RadioButton` components, but I believe it's something that should have been there

Reply:
probably because there has been no real use case until now